refinery/analysis_manager/models.py

```diff
@@ -29,6 +29,10 @@ class AnalysisStatus(models.Model):
     #: state of Galaxy history
     galaxy_history_state = CharField(max_length=10, blank=True,
                                      choices=GALAXY_HISTORY_STATES)
+
+    #: percentage of successfully imported datasets in Galaxy history
+    galaxy_import_progress = PositiveSmallIntegerField(default=0)
```

Comment:
Is there are reason that this is `default=0` and just below it's `blank=True, null=True`?